letsencrypt/cli.py

```diff
@@ -845,14 +845,17 @@ def _handle_exception(exc_type, exc_value, trace, args):
 
         if issubclass(exc_type, errors.Error):
             sys.exit(exc_value)
-        elif args is None:
-            sys.exit(
-                "An unexpected error occurred. Please see the logfile '{0}' "
-                "for more details.".format(logfile))
         else:
-            sys.exit(
-                "An unexpected error occurred. Please see the logfiles in {0} "
-                "for more details.".format(args.logs_dir))
+            # Tell the user a bit about what happened, without overwhelming
+            # them with a full traceback
+            msg = ("An unexpected error occurred.\n" +
+                   traceback.format_exception_only(exc_type, exc_value)[0] +
```

Comment:
There are some cases, like permissions errors, which are end-user-actionable.  There are others which are not. In the long run we can try to exhaustively classify and which ones to who, but this PR is trying to get a more reasonable midpoint between showing everything and hiding everything.  The "unexpected error" clarification is a reasonable warning before the user sees something cryptic and possibly nonsensical.